pkg/systemd/overview-cards/uncleanShutdownStatus.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { Button } from "@patternfly/react-core/dist/esm/components/Button/index.js";
+import { Flex, FlexItem } from "@patternfly/react-core/dist/esm/layouts/Flex/index.js";
+import { ExclamationTriangleIcon} from "@patternfly/react-icons";
+
+import cockpit from "cockpit";
+
+import './uncleanShutdownStatus.scss';
+
+const _ = cockpit.gettext;
+
+export const UncleanShutdownStatus = () => {
+    const [uncleanShutdownId, setUncleanShutdownId] = useState(() => {
+        cockpit.spawn(
+            ["last", "-xn2", "shutdown", "reboot"],
+            { err: "ignore", }
+        ).then((data) => {
+            const shutdownLines = data
+                .split("\n")
+                .filter((line) => line.startsWith("shutdown"));
+
+            const newUncleanShutdownId = shutdownLines.length == 0 ? data : "";
+            setUncleanShutdownId(newUncleanShutdownId);
+            setUncleanShutdownStatusVisible(
+                newUncleanShutdownId != cockpit.localStorage.getItem("dismissed-unclean-shutdown-id")
+            );
+        });
```

Comment:
Please use `useEffect` or better `useInit` for setting the initial state, combining this in `useState` a bit hard to read.